facade-charm/README.md

```diff
@@ -0,0 +1,26 @@
+# Facade charm
+
+Usage:
+`tox -e pack`
+`juju deploy facade`
+`juju relate facade:provide-your-interface <your charm>`
```

Comment:
```suggestion
`juju integrate facade:provide-your-interface <your charm>`
```

:smile:

Reply:
"tell me how long you've been at canonical without telling me how long you've been at canonical"